frames/src/main/res/values/strings.xml

```diff
@@ -30,7 +30,6 @@
     <!--Billing form screen-->
     <string name="cko_billing_form_screen_title" translatable="true">Billing address</string>
     <string name="cko_billing_form_button_save" translatable="true">Save</string>
-    <string name="cko_billing_form_input_field_full_name" translatable="true">Full name</string>
```

Comment:
Why was it removed?

Reply:
We do not require it. We need to discuss this field. Currently, in the IOS design, it is not there.